src/About/style.scss

```diff
@@ -0,0 +1,96 @@
+@import './_variables.scss';
+
+.wrapper {
+  background-attachment: fixed;
+  background-image: url('../image/bg_about.png');
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.section-about {
+  display: flex;
+  width: 70%;
+  padding: 60px 0 20px;
+  margin: 0 auto;
+}
+
+.about-right {
+  margin-left: 5%;
+}
+
+.about-left {
+  margin-top: 35px;
+  margin-left: 11%;
+
+
+  h1 {
+    padding-bottom: 13px;
+    font-size: 36px;
```

Comment:
it is better to use rem for font size. Then for mobile it will look better

Reply:
ok, it  was changed